spec/requests/customers_spec.rb

```diff
@@ -0,0 +1,84 @@
+require 'rails_helper'
+
+RSpec.describe "Customers", type: :request do
```

Comment:
the 1st Rspec block should be removed

Reply:
What part needs to be removed? I don't know what lines of code you mean.

Thanks for your time!